lib/src/model/annotation.dart

```diff
@@ -74,4 +74,7 @@ class Annotation extends Feature with ModelBuilder {
 
   @override
   int get hashCode => annotation.hashCode;
+
+  @override
+  String get cssClassName => 'annotation';
```

Comment:
If we have no CSS rules associated with this class name, it would be better to just leave it blank.